src/project/qml/MuseScore/Project/AudioGenerationSettings.qml

```diff
@@ -146,23 +167,31 @@ RadioButtonGroup {
             }
 
             IncrementalPropertyControl {
-                width: 96
+                Layout.preferredWidth: numberOfSavesItem.textPart2InSpinbox ? 96 : 60
 
                 minValue: 2
                 maxValue: 30
                 currentValue: settingsModel.numberOfSaves
-                measureUnitsSymbol: qsTrc("project/save", "Saves")
+                measureUnitsSymbol: numberOfSavesItem.textPart2InSpinbox ? numberOfSavesItem.textPart2 : ""
                 step: 1
                 decimals: 0
 
                 navigation.panel: root.navigationPanel
-                navigation.row: index
+                navigation.row: numberOfSavesItem.index
                 navigation.column: 1
+                navigation.accessible.name: qsTrc("project/save", "Every N saves") + ": " + currentValue
```

Comment:
@shoogle / @Eism What do you think about the accessibility descriptions? I realized that I should maybe do it slightly different than what I'm doing now:
- the radio button gets "Every N saves" as its accessibility name
- the number input field gets just its value as its name. 

Is that a good idea? Or do you prefer another option?

Reply:
You wouldn't normally make the value part of the name unless you're doing some kind of hack. The screen reader should report the value separately to the name.

For example, you might hear the screen reader say _"Visible checkbox not checked"_. The name of the checkbox is just "Visible". The screen reader knows that it has to tell you the role ("checkbox") and the value ("not checked") so you don't need to include that information in the name.

 You'd probably need to use "Every N saves" as the name for both the radio button and the input field (or the input field could be called something like "Saves before regenerating audio"). The screen reader should announce the value separately to this, but you'd need to test to confirm. There are instructions for how to enable your build-in screen reader [here](https://github.com/musescore/MuseScore/wiki/Accessibility-Testing#popular-screen-readers).

Reply:
I now made it "Every N saves" for both the radio button and the spin box.

We might want to refine the way screen readers treat our spinboxes; currently, they treat them as plain text fields. This was better in MS3.